core/shared/src/main/scala/zio/ZLayer.scala

```diff
@@ -58,7 +58,7 @@ sealed abstract class ZLayer[-RIn, +E, +ROut] { self =>
    */
   final def ++[E1 >: E, RIn2, ROut1 >: ROut, ROut2](
     that: ZLayer[RIn2, E1, ROut2]
-  )(implicit ev: Has.Union[ROut1, ROut2], tag: Tag[ROut2]): ZLayer[RIn with RIn2, E1, ROut1 with ROut2] =
+  )(implicit ev: Has.Union[ROut1, ROut2]): ZLayer[RIn with RIn2, E1, ROut1 with ROut2] =
```

Comment:
You need to add another method with the old JVM signature to appease MiMa

Reply:
Thanks!

Instead I introduced new method concat to make mima happier and for backward compatibility